shopfloor/actions/inventory.py

```diff
@@ -51,8 +51,10 @@ def _get_existing_quant(self, location, product, package=None, lot=None, limit=1
             domain.append(("lot_id", "=", False))
         return self.inventory_model.search(domain, limit=limit)
 
-    def _create_draft_inventory(self, location, product, lot=None):
-        quants = self._get_existing_quant(location, product, lot=lot, limit=None)
+    def _create_draft_inventory(self, location, product, package=None, lot=None):
```

Comment:
This should be completed with owner_id field too.

Reply:
maybe after the forward port from  v14